src/docfx/build/xref/XrefMap.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    public class XrefMap
+    {
+        // TODO: key could be uid+moniker+locale
+        private readonly IReadOnlyDictionary<string, XrefSpec> _map;
```

Comment:
>IReadOnlyDictionary [](start = 25, length = 19)

consider case sensitivity?

Reply:
it should be case sensitive by default, right?

---
In reply to: [218288055](https://github.com/dotnet/docfx/pull/3428#discussion_r218288055) [](ancestors = 218288055)

Reply:
given uid is mostly used to identify reference language elements, and most programming languages are case sensitive, it should be case sensitive. What is the behavior in v2?

Reply:
it's case sensitive in v2, as it relies on xref service, which is case sensitive

---
In reply to: [218298652](https://github.com/dotnet/docfx/pull/3428#discussion_r218298652) [](ancestors = 218298652)